sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/RecognizeCustomForms.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer;
+
+import com.azure.ai.formrecognizer.models.OperationResult;
+import com.azure.ai.formrecognizer.models.RecognizedForm;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.util.IterableStream;
+import com.azure.core.util.polling.SyncPoller;
+
+/*
+ *  This sample demonstrates how to analyze a form from a document with a custom
```

Comment:
I think you want to link to the labeled and unlabeled custom model training samples